server/google_apis/mono_pkg.yaml

```diff
@@ -0,0 +1,12 @@
+dart:
+  - dev
+  - 2.12.0
+
+stages:
+  - analyze:
+    - dartanalyzer: --fatal-infos .
```

Comment:
Doesn't mono_repo support generating these with the `dart` commands?

Reply:
It's complicated.

This syntax is specific to `mono_repo`, which started out only supporting travis ... which was based on the old way of doing things.

No high on my list to fix.